python/lsst/daf/butler/core/formatter.py

```diff
@@ -77,6 +82,11 @@ def fileDescriptor(self) -> FileDescriptor:
         (`FileDescriptor`, read-only)"""
         return self._fileDescriptor
 
+    @property
+    def dataId(self) -> Dict:
+        """DataId associated with this formatter (`dict`)"""
```

Comment:
If constructor takes a `DataCoordinate`, that's what the type is going to be here, too, right?

Reply:
Yes, I should have gone back and fixed that